lib/lightning/pipeline.ex

```diff
@@ -28,6 +28,11 @@ defmodule Lightning.Pipeline do
     |> process()
   end
 
+  # @impl Oban.Worker
+  # def timeout(_job),
+  #   do:
+  #     Application.get_env(:lightning, :max_run_duration)
+
```

Comment:
Why is this commented out? Where else is the timeout set?

Reply:
@stuartc I introduced this to be able generate timeout exceptions and handle them in the `handle_event([:oban, :job, :exception], measure, meta, _pid)` in the Oban manager but then I could test that the exception is fired but I completely couldn't capture it in the Oban manager. I left it commented out to get your attention and pair on it during our 1-1. This is very related to your comment [here](https://github.com/OpenFn/Lightning/pull/495#discussion_r1045711919).

Reply:
All good now